packages/reason-parser/reason-parser.2.0.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "Jordan Walke <jordojw@gmail.com>"
+authors: "Jordan Walke <jordojw@gmail.com>"
+homepage: "https://github.com/facebook/reason"
+bug-reports: "https://github.com/facebook/reason/issues"
+license: "BSD"
```

Comment:
Btw. [here's](https://github.com/ocaml/opam/pull/2224) the discussion about the license field and spdx.